cmd/client/commandv2/edit.go

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package commandv2
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/megaease/easegress/v2/cmd/client/general"
+	"github.com/megaease/easegress/v2/cmd/client/resources"
+	"github.com/spf13/cobra"
+)
+
+// EditCmd returns edit command.
+func EditCmd() *cobra.Command {
+	examples := []general.Example{
+		{Desc: "Edit a resource with name", Command: "egctl edit <resource> <name>"},
```

Comment:
Could we specify editor like vim, nano.

Reply:
Yes, in `egctl-cheat-sheet.md`, mentioned use of env variable of `EGCTL_EDITOR` to change editor. I will add an example about that in examples.